apps/evm/src/pages/Proposal/Commands/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Card, type CardProps } from 'components';
+import { useTranslation } from 'libs/translations';
+import { Progress } from './Progress';
+
+export interface CommandsProps extends CardProps {}
+
+export const Commands: React.FC<CommandsProps> = ({ ...otherProps }) => {
+  const { t } = useTranslation();
+
+  // TODO: fetch (see VEN-2701)
+  const successfulPayloadsCount = 4;
+  const totalPayloadsCount = 5;
```

Comment:
Wouldn't these be props passed to this component?

Reply:
I'm not sure yet. The idea I had was to fetch them from the component directly, since we use React Query it wouldn't generate an additional query if it was fetched from the Proposal page too.